### ISSUE TITLE
Force conversion of all fields to strings

### ISSUE THREAD

**Issue by lejacobroy** [mrodrig/json-2-csv#149]
## Background Information
- Module Version: `3.7.0`
- Node/Browser Version: `13.13.0 `

The issue I'm reporting is with:
- [ ] json2csv
- [x] csv2json

I have...
- [x] searched to see if an issue has already been reported.
- [x] verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
- [x] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).

## Expected Behavior
I have a csv with some rows that contains the string "39e1804". I want it to be a string when converted to json.

## Actual Behavior
The csv string "39e1804" gets read as a number in scientific representation, and converted to the number "Infinity". Even if I use toString(), it reports the string "Infinity", since that's the number converted to string.

## Data Sample 
My csv gets read as a buffer, converted to string and then sent to csv2json.
CSV:
```csv
"code1,code2
test,39e1804"
```

(or)

JSON:
```json

```

## Code Example
```javascript
// Please include a simple example to replicate the issue
let converter = require('json-2-csv');

const options = {
                delimiter: {
                  eol: '\r\n'
                },
                excelBOM: true,
                excelStrings: true
              };
let final_json = await converter.csv2jsonAsync(file.data.toString(), options)
```

**Comment by mrodrig** [comment]
Thanks for reporting this @lejacobroy! Having it converted to `Infinity` is definitely not the desired behavior. I added logic to ensure this shouldn't happen for values of that format again and will be releasing the fix as part of `3.7.1` shortly. Thanks again for your report.

Have a nice weekend and stay safe!

**Comment by lejacobroy** [comment]
Thank you for the fast reply and fix! I'll test it this week.
